src/Symfony/Component/Translation/Bridge/Loco/Tests/LocoProviderFactoryTest.php

```diff
@@ -34,6 +34,6 @@ public function incompleteDsnProvider(): iterable
 
     public function createFactory(): ProviderFactoryInterface
     {
-        return new LocoProviderFactory($this->getClient(), $this->getLogger(), $this->getDefaultLocale(), $this->getLoader());
+        return new LocoProviderFactory($this->getClient(), $this->getLogger(), $this->getDefaultLocale(), $this->getLoader(), $this->getTranslatorBag());
```

Comment:
At least one test should keep using the old argument set and mark it as group legacy.

Reply:
I've added 2 test cases `LocoProviderFactoryWithoutTranslatorBagTest` and `LocoProviderWithoutTranslatorBagTest` which extends respectively `LocoProviderFactoryTest` and `LocoProviderTest` test cases, but creating a `LocoProviderFactory`/`LocoProvider` instance when `$translatorBag` is null.

I've also adapted the `LocoProviderWithoutTranslatorBagTest#testReadWithLastModified` test to ensure nothing goes wierd.

Thanks for your review!